l10n_br_sale_stock/test/sale_order_demo.yml

```diff
@@ -62,7 +62,6 @@
     assert sale_order.picking_ids, "Delivery order is not created."
     for picking in sale_order.picking_ids:
       assert picking.state == "auto" or "confirmed", "Delivery order should be in 'Waitting Availability' state."
-      assert picking.type == 'out',"Shipment should be Outgoing."
```

Comment:
Should be: 
assert picking.picking_type_id.code == 'outgoing',"Shipment should be Outgoing."